cristin-commons/src/main/java/no/unit/nva/cristin/facet/CristinFacetUriParamAppender.java

```diff
@@ -12,7 +12,7 @@
 
 public class CristinFacetUriParamAppender {
 
-    public static final String PARAM_VALUE_DELIMITER = ",";
+    public static final String PARAM_VALUE_DELIMITER = ",|%2C";
```

Comment:
PARAM_VALUE_DELIMITER is also defined other places in the codebase, concider to merge them.